internal/commands/build.go

```diff
@@ -128,3 +166,20 @@ func addEnvVar(env map[string]string, item string) map[string]string {
 	}
 	return env
 }
+
+func parseProjectToml(descriptorPath string) (project.Descriptor, string, error) {
+	actualDescriptorPath := descriptorPath
+	if descriptorPath == "" {
+		actualDescriptorPath = "project.toml"
```

Comment:
I suspect this needs to be relative to the app path.

Reply:
I must have removed it during a refactor.